src/codes.ts

```diff
@@ -43,6 +45,23 @@ const codeList: ReadonlyArray<Readonly<ICode>> = Object.freeze((
 
 export const ChainType = tcomb.irreducible('ChainType', (n) => isValid(n));
 
+const forCode = (c: ChainCode): UnspentType => {
```

Comment:
I would name this function `typeForCode`

Reply:
Done